app/views/layouts/base.html.erb

```diff
@@ -71,6 +71,7 @@ See COPYRIGHT and LICENSE files for more details.
         </h1>
         <main-menu-toggle></main-menu-toggle>
         <a href="" class="hidden-for-sighted skip-navigation-link"
+           id="skip-navigation--content"
```

Comment:
This ID got lost a few years back, breaking the "Skip to content" hidden link when tabbing through the menu